src/metrics_server/test_integration_action.sh

```diff
@@ -85,14 +85,14 @@ EOF
 
 echo "Connections request:"
 cat $CONNECTIONS_REQUEST
-curl -X POST -H "Content-Type: application/json" -d @$CONNECTIONS_REQUEST $METRICS_URL/connections && echo
+curl -X POST -H "Content-Type: application/json" -d @$CONNECTIONS_REQUEST $METRICS_URL/$CONNECTIONS_PATH && echo
```

Comment:
The FEATURES_PATH and CONNECTIONS_PATH seem pointless and hurt readability. Can you remove them and write the path directly?
Or perhaps @alalamav wants to keep them?

Reply:
Removing sounds fine to me.

Reply:
Done